src/scripts/Frame.js

```diff
@@ -0,0 +1,31 @@
+function Frame() {
+  this.rollsRemaining = 2;
+  this.totalScoreRecord = [];
+}
+
+Frame.prototype.registerRoll = function(pinsKnockedOver) {
+  if (pinsKnockedOver === 10) {
```

Comment:
Condition could be extracted to function. As is it's not clear what this means in terms of the game of bowling